packages/@aws-cdk/aws-ecs/test/fargate/test.fargate-service.ts

```diff
@@ -254,6 +254,12 @@ export = {
         }
       }));
 
+      expect(stack).to(haveResource('AWS::ECS::Service', {
+        // if any load balancer is configured and healthCheckGracePeriodSeconds is not
```

Comment:
Where is the case where the default is not applied?

Reply:
At the top of `test.ec2-service.ts` and `test.fargate-service.ts`, there are 2 existing test cases where LB are not set, hence the `healthCheckGracePeriodSeconds` is also not set.

There are 2 additional test cases that check the `healthCheckGracePeriodSeconds` is respected when set.

I did not change those because they seem sufficient to cover the other cases (beside the one tested here).